pkg/xray/policies.go

```diff
@@ -109,7 +109,7 @@ var getPolicySchema = func(criteriaSchema map[string]*schema.Schema, actionsSche
 				Type:             schema.TypeString,
 				Required:         true,
 				Description:      "Type of the policy",
-				ValidateDiagFunc: validator.StringInSlice(true, "Security", "License", "Operational_Risk"),
+				ValidateDiagFunc: validator.StringInSlice(false, "security", "license", "operational_risk"),
```

Comment:
We should retain the case-sensitive check flag, because if the configuration has mixed case string then the API will fail.

Reply:
The flag is `ignoreCase`, meaning if it's `false` then it's case-sensitive.